changelog.txt

```diff
@@ -1 +1,2 @@
+feature - Adds functions.app() api to access the app instance used by functions
```

Comment:
this should be `functions.apps()` right? same with your commit desc / sample

Reply:
nope, its exported as app to match the api that was approved